Configuration/Geometry/README.md

```diff
@@ -12,6 +12,7 @@ Tracker:
 Calorimeters:
 * C3: HGCal (v8) + Phase2 HCAL and EB
 * C4  HGCal (v9) + Phase2 HCAL and EB
+* C5  HGCal (v9) + Phase2 HCAL and EB (without virtual wafers)
```

Comment:
`C5: HGCal (v9, without virtual wafers) + Phase2 HCAL and EB`

Reply:
Done